src/Nox.Types/Types/Color/Color.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Color"/> type and value object. 
+/// </summary>
+public class Color : ValueObject<byte[], Color>
+{
+    private byte _alpha { get; set; }
+    private byte _red { get; set; }
+    private byte _green { get; set; }
+    private byte _blue { get; set; }
+    
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object.
+    /// </summary>
+    public Color()
+    {
+        Value = new byte[4];
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object with empty color.
+    /// </summary>
+    public static Color Empty
+    {
+        get => new();
+    }
+
+    /// <summary>
+    /// Validates a <see cref="Color"/> object.
+    /// </summary>
+    /// <returns>true if the <see cref="Color"/> value is valid.</returns>
+    internal override ValidationResult Validate()
+    {
+        var result = new ValidationResult();
+
+        if (Value.Length < 1)
+            result.Errors.Add(new ValidationFailure(nameof(Value),
+                $"Could not create a Nox {nameof(Color)} type as an empty {nameof(Value)} is not allowed."));
+
+        if (Value.Length > 4)
+            result.Errors.Add(new ValidationFailure(nameof(Value),
+                $"Could not create a Nox {nameof(Color)} type with more than 4 values."));
+
+        return result;
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object.
+    /// </summary>
+    /// <param name="red">The red color.</param>
+    /// <param name="green">The green color.</param>
+    /// <param name="blue">The blue color.</param>
+    /// <exception cref="TypeValidationException"></exception>
+    public static Color From(byte red, byte green, byte blue)
+    {
+        var color = System.Drawing.Color.FromArgb(byte.MaxValue, red, green, blue);
+
+        return CreateColor(color);
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object.
+    /// </summary>
+    /// <param name="alpha">The alpha color.</param>
+    /// <param name="red">The red color.</param>
+    /// <param name="green">The green color.</param>
+    /// <param name="blue">The blue color.</param>
+    /// <exception cref="TypeValidationException"></exception>
+    public static Color FromRgba(byte alpha, byte red, byte green, byte blue)
```

Comment:
Rename the method from "FromRgba" to "From"

Reply:
Done